### PR TITLE
[TTAHUB-831] Participants Filter

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Added the ability to filter by Participants on both the AR and Regional Dashboard.

## How to test

Filter by participants on both the AR landing and regional dashboard pages. 

Both the recipient and other entity sections should be divided by text shown in:

https://www.figma.com/proto/sj6v77fW5VHnmGdih0m2Ph/TTAHUB-688?node-id=325%3A11669&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=325%3A11669&show-proto-sidebar=1

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-831

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/Constants.js [hunk 1]
```diff
@@ -253,3 +253,42 @@ export const GOAL_SUSPEND_REASONS = [
   'Recipient is not responding',
   'Regional Office request',
 ];
+
+/*
+  Please keep in sync with:
+  frontend > src > pages > ActivityReport > constants.js
+  src > constants.js
+*/
```
Comment (thewatermethod) [inline frontend/src/Constants.js:261]: 🔥

File: src/scopes/activityReport/participants.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import filterArray from './utils';
+import {
+  ALL_PARTICIPANTS,
+} from '../../constants';
+
+function onlyValidParticipants(query) {
+  if (!Array.isArray(query)) {
+    return [query].filter((reason) => ALL_PARTICIPANTS.includes(reason));
+  }
```
Comment (thewatermethod) [inline src/scopes/activityReport/participants.js:9]: 😃 might want to change that iterator from ```reason``` to ```participant```

**Comment by thewatermethod** [comment]
Only comment beyond what I said above is that the disabled divider text is smaller in the mockup (14px) than the regular options. I think other than that, looks good to go

**Review by thewatermethod — APPROVED** [review]

File: src/scopes/activityReport/participants.js (hunk 2 above)
Comment (AdamAdHocTeam) [reply src/scopes/activityReport/participants.js:9]: Nice catch.

**Pull request merged**